### PR TITLE
Build dashboard route

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

**Comment by baomastr** [comment]
we need one more ticket for fixing responsiveness of tables that displaying supply and borrow markets data to solve the issue with horizontal scrolling

File: src/pages/Dashboard/index.tsx [hunk 1]
```diff
@@ -1,19 +1,72 @@
...
+      },
+      {
+        title: t('dashboard.markets.tabBorrow'),
+        content: <BorrowMarket css={styles.item} />,
+      },
+    ],
+    [],
+  );
+
+  const renderMarkets = () => {
```
Comment (coreyar) [inline src/pages/Dashboard/index.tsx]: Functions labeled as render can usually be better written as a component with props

File: src/pages/Dashboard/index.tsx [hunk 2]
```diff
@@ -1,19 +1,72 @@
...
+    if (isSmDown) {
+      return (
+        <div css={styles.tabsWrapper}>
+          <h4 css={[styles.tabsHeader, styles.tabsTitle]}>{t('dashboard.markets.title')}</h4>
+          <Tabs css={styles.tabsHeader} tabsContent={tabsContent} />
+        </div>
+      );
+    }
+
+    if (isLgDown) {
```
Comment (coreyar) [inline src/pages/Dashboard/index.tsx]: Was there a reason that prevented us from doing this in css?

Comment (baomastr) [reply src/pages/Dashboard/index.tsx]: more clear responsive cases

Comment (baomastr) [reply src/pages/Dashboard/index.tsx]: there are 3 different cases for mobile/tablets/desktop with different layouts

File: src/pages/Dashboard/index.tsx (hunk 1 above)
Comment (baomastr) [reply src/pages/Dashboard/index.tsx]: refactored

File: src/pages/Dashboard/index.tsx (hunk 2 above)
Comment (baomastr) [reply src/pages/Dashboard/index.tsx]: also refactored a bit

**Comment by baomastr** [comment]
horizontal scrolling fix resolved in https://github.com/VenusProtocol/venus-protocol-interface/pull/356

**Comment by baomastr** [comment]
https://github.com/VenusProtocol/venus-protocol-interface/pull/361

**Pull request closed without merging**